src/service/CodeExecutionInfo.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, createContext, useState } from "react";
+import { CodeExecutionInfoType } from "types";
+
+const CodeExecutionInfoContext = createContext<CodeExecutionInfoType | null>(null);
+
+const CodeExecutionInfoProvider: FC<React.ReactNode> = ({ children }) => {
```

Comment:
Make FC as `React.FC` because I want to remain consistent with how the code is written in other files.

Reply:
Made requested changes.